packages/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Why do we need this file?

Reply:
Moar parserOptions. Remove it and things go sads.

Reply:
Ok. I have another approach I can try once the dust settles.